chore: remove zenhub from release process (#25701)
cypress-io/cypress: guides/release-process.md

    ```

- Set up the following environment variables:
  - For the `release-automations` step, you will need setup the following envs:
    - GitHub token - Found in 1Password.
    - The `cypress-bot` GitHub app credentials. Found in 1Password.
    ```text
    GITHUB_TOKEN="..."
    GITHUB_APP_CYPRESS_INSTALLATION_ID=
    GITHUB_APP_ID=
    GITHUB_PRIVATE_KEY=
    ```

>   - For purging the Cloudflare cache (need for the `prepare-release-artifacts` script in step 6), you'll need `CF_ZONEID` and `CF_TOKEN` set. These can be found in 1Password.

Reviewer: ```suggestion
  - For purging the Cloudflare cache (needed for the `prepare-release-artifacts` script in step 6), you'll need `CF_ZONEID` and `CF_TOKEN` set. These can be found in 1Password.
```

        - [cypress-realworld-app](https://github.com/cypress-io/cypress-realworld-app) uses yarn and represents a typical consumer implementation.
    - Optionally, do more thorough tests, for example test the new version of Cypress against the Cypress Cloud repo.

2. Ensure all changes to the links manifest to [`on.cypress.io`](https://github.com/cypress-io/cypress-services/tree/develop/packages/on) have been merged to `develop` and deployed.

3. Create a Release PR Bump, submit, get approvals on, and merge a new PR. This PR Should:
    - Bump the Cypress `version` in [`package.json`](package.json)
    - Bump the [`packages/example`](../packages/example) dependency if there is a new [`cypress-example-kitchensink`](https://github.com/cypress-io/cypress-example-kitchensink/releases) version
    - Follow the writing the [Cypress Changelog release steps](./writing-the-cypress-changelog.md#release) to update the [`cli/CHANGELOG.md`](../cli/CHANGELOG.md).

> 4. Once the `develop` branch is passing in CI and the and you have confirmed the `cypress-bot` has comment on the commit with the pre-release versions for `darwin-x64`, `darwin-arm64`, `linux-x64`,`linux-arm64`, and `win32-x64`, publishing can proceed.

Reviewer: ```suggestion
4. Once the `develop` branch is passing in CI and you have confirmed the `cypress-bot` has commented on the commit with the pre-release versions for `darwin-x64`, `darwin-arm64`, `linux-x64`,`linux-arm64`, and `win32-x64`, publishing can proceed.
```

    yarn binary-release --version X.Y.Z
    ```

15. Merge the documentation PR from step 11 and the new docker image PR created in step 12 to release the image.

16. If needed, deploy the updated [`cypress-example-kitchensink`][cypress-example-kitchensink] to `example.cypress.io` by following [these instructions under "Deployment"](../packages/example/README.md).

17. Once the release is complete, create a Github tag off of the release commit which bumped the version:
    ```shell
    git checkout develop
    git pull origin develop
    git log --pretty=oneline
    # copy sha of the version bump commit
    git tag -a vX.Y.Z -m vX.Y.Z <sha>
    git push origin vX.Y.Z
    ```

18. Create a new [GitHub release](https://github.com/cypress-io/cypress/releases). Choose the tag you created previously and add contents to match previous releases.

> 19. Add a comment to each GH issue that has been resolved with the new published version. Download the `releaseData.json` from CI and run the following command inside of [cypress-io/release-automations][release-automations]:

Author: ```suggestion
19. Add a comment to each GH issue that has been resolved with the new published version. Download the `releaseData.json` artifact from the `verify-release-readiness` CircleCI job and run the following command inside of [cypress-io/release-automations][release-automations]:
```

> 19. Add a comment to each GH issue that has been resolved with the new published version. Download the `releaseData.json` from from CI and run the following command inside of [cypress-io/release-automations][release-automations]:

Reviewer 2: Can we specify where to find this file in CI?

Author: done: [`57647c9` (#25701)](https://github.com/cypress-io/cypress/pull/25701/commits/57647c9be9e5a0951487aca2d7da991add7be6b4)

> 19. Add a comment to each GH issue that has been resolved with the new published version. Download the `releaseData.json` from from CI and run the following command inside of [cypress-io/release-automations][release-automations]:

Reviewer: ```suggestion
19. Add a comment to each GH issue that has been resolved with the new published version. Download the `releaseData.json` from CI and run the following command inside of [cypress-io/release-automations][release-automations]:
```

> 19. Add a comment to each GH issue that has been resolved with the new published version. Download the `releaseData.json` from from CI and run the following command inside of [cypress-io/release-automations][release-automations]:

Reviewer: Can the `release-automations` script just download the file?

Author: I didn't put the effort into this because this will eventually all run in the same release workflow in CircleCi and I don't anticipate this step being around for long.
